lib/logstash/filters/json.rb

```diff
@@ -83,7 +83,7 @@ def filter(event)
     end
 
     if @target
-      event.set(@target, parsed)
+      event.set(event.sprintf(@target), parsed)
```

Comment:
There's room for confusion here that a user may want a value from the `parsed` object to dictate what field is set. If we're going to allow this behavior, I think we need to document what does and doesn't work for this plugin. Thoughts?